raiden/ui/cli.py

```diff
@@ -38,7 +39,6 @@
 from raiden.network.matrixtransport import MatrixTransport
 from raiden.network.transport.udp.udp_transport import UDPTransport
 from raiden.network.rpc.client import JSONRPCClient
-from raiden.network.sockfactory import SocketFactory
```

Comment:
I'd rather we keep this import in place and import `mirakuru.HTTPExecutor` here as well. Importing them repeatedly in `if` blocks across the code looks ugly and will not make a big difference in performance.